lib/soapy_cake/helper.rb

```diff
@@ -23,17 +23,21 @@ def require_params(opts, params)
       end
     end
 
-    def translate_booleans!(opts)
+    def translate_booleans(opts)
+      opts = opts.dup
       opts.each do |k, v|
```

Comment:
You can move this into one line `opts.dup.each ...`and then get rid of the last line.